### DISCUSSION TITLE
404 - Not working with Locales?

### DISCUSSION THREAD

**Discussion by jonny-dengro** [opcodesio/log-viewer#49] [Q&A] [answered]
I get a 404 error when going to https://my-app.test/log-viewer.

We use locales in our URLs so I updated the config as below:
` 'route_path' => 'en-GB/log-viewer',`

This gives me access to the dashboard via https://my-app.test/en-GB/log-viewer.

I can download the log file, however a 404 is thrown when I try the bleow actions:
- view on log file from the Dashboard 
- clear cache
- delete
- click 'shorter stack traces' in the settings menu.

Is there a known workaround for this?

**Comment by arukompas** [comment, marked as answer]
@jonny-dengro hey!

If your app is accessible under a certain subdirectory only, then you'll also need to update Livewire configuration, because that's a dependency for the Log Viewer and be default it loads its resources from the root path.

Check this answer for more information - https://github.com/opcodesio/log-viewer/issues/8#issuecomment-1223563122

Let me know if you still have questions!
